Tests.SimpleConfigSections/ConfigurationSectionBasedOnClassSpecs.cs

```diff
@@ -64,16 +84,41 @@ public override object  ConvertFrom(ITypeDescriptorContext context, System.Globa
         }
     }
 
+    public enum EnumWithDefault
+    {
+        Default = 0,
+        Other
+    }
+
+    public enum EnumWithNoDefault
+    {
+        First = 1,
+        Second = 2
+    }
+
     public class SimpleConfiguration
     {
-		[Display(Name = "name")]
+        [Display(Name = "name")]
         public virtual string Name { get; set; }
         public virtual int Count { get; set; }
 
         public virtual SimpleConfiguration Child { get; set; }
 
         public virtual Regex Matcher { get; set; }
 
+        public virtual EnumWithDefault Enum1 { get; set; }
+        [Default(DefaultValue = EnumWithDefault.Other)]
+        public virtual EnumWithDefault Enum2 { get; set; }
+
+        public virtual EnumWithNoDefault Enum3 { get; set; }
+        [Default(DefaultValue = EnumWithNoDefault.Second)]
+        public virtual EnumWithNoDefault Enum4 { get; set; }

```

Comment:
That's cause I am trying to test 4 different cases:

1. A property of type Enum where this enum has a default-value (ie. a value with int == 0).
2. A property of type Enum where this enum has a default-value (ie. a value with int == 0), which has been decorated with Default attribute pointing to a value other than the default one.
3. A property of type Enum where this enum _does not_ have a default-value (ie. a value with int == 0).
3. A property of type Enum where this enum _does not_ have a default-value (ie. a value with int == 0), which has been decorated with Default attribute pointing to an especific value.

In all cases, this tests are supposed to evaluate a situation where this enum properties have not value defined at app.config.